vello_tests/src/lib.rs

```diff
@@ -60,7 +60,7 @@ pub use snapshot::{
 pub struct TestParams {
     pub width: u32,
     pub height: u32,
-    pub base_colour: Option<Color>,
+    pub base_color: Option<Color>,
```

Comment:
Perhaps we could go the other way and introduce more diverse spellings of the word color, instead of stamping them out.
```suggestion
    /// La couleur d'arrière-plan, le cas échéant.
    pub base_colour: Option<Color>,
```